pangeo-notebook/conda-linux-64.lock

```diff
@@ -611,3 +614,7 @@ https://conda.anaconda.org/conda-forge/noarch/ipyurl-0.1.2-pyh3684270_1.tar.bz2#
 https://conda.anaconda.org/conda-forge/noarch/jupyterhub-singleuser-2.1.1-hd8ed1ab_0.tar.bz2#86aab26e793c1527e5240dfb08fc08aa
 https://conda.anaconda.org/conda-forge/noarch/pangeo-notebook-2022.01.31-hd8ed1ab_0.tar.bz2#ccd7fecfc6a66582b7269b3f1940d883
 https://conda.anaconda.org/conda-forge/noarch/xarray_leaflet-0.1.15-pyhd8ed1ab_0.tar.bz2#33925089afaae43154afe188bdbe0670
+https://conda.anaconda.org/conda-forge/noarch/argopy-0.1.6-pyhd8ed1ab_1.tar.bz2#54e0baf97064216935453ee380ec520d
```

Comment:
0.1.6 is also getting installed in the regular pangeo-notebook image. I'm not sure why. Perhaps you want to pin a version >= 0.1.9. Although I don't understand why it doesn't pick the latest version automatically.